Update navigation paths for editor sidebar and Projects page (#189)
SubframeApp/subframe: packages/docs/learn/projects/overview.mdx

## Create a project

> 1. Open the **Projects** page — click your avatar in the top right and select **Projects**

Author: ```suggestion
1. Open the **Projects** page — click the Subframe logo in the top left
```
